infrastructure/transform/feature-data-store.tf

```diff
@@ -128,6 +128,7 @@ resource "null_resource" "create_sql_user" {
   # load a csv file into a new SQL table
   provisioner "local-exec" {
     command = <<EOF
+      set -o errexit
```

Comment:
[Thinking Out Loud]

This clearly works. I'm just not sure why/how it's required.

I'm assuming this is because of the use of `<<EOF`, and that if we took those three lines of bash and wrapped them up into a single script we'd probably be fine (so long as that single script raised errors correctly).

I wonder if this (using `EOF`) is a pattern we should avoid in the future, and a single script (with proper error handling) should be written to wrap such blocks up.

There's one other use of `EOF` and `local-exec` which, I assume, also has the potential to fail silently. I don't suggest we fix this, but it may want a bug raising. What do you think?

Reply:
For posterity: this has to do not with <<EOF but with having multiple commands in a local-exec block, as shell will exit with the exit code of the last command. In this case, we want it to fail if one of the commands fails 

I have checked and there aren't currently any other local-exec blocks with multiple commands in them

Reply:
Turns out it's not exactly like that, the `set -o errexit` flag is needed specifically because of the use of the retry.sh script. However, the outcome remains the same: we do not need to change other parts of the code as they don't have this pattern 

In the future, we should err on the side of creating a separate script for any local-exec longer than a couple of lines.